### PR TITLE
Feature/configure eslint

### LINKED ISSUE #8: Configure eslint

### PR DESCRIPTION
**Pull request by SpiraTecGabelM**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -7,65 +7,65 @@ import {
...
+                    {darkMode ? 'Switch to Light Mode' : 'Switch to Dark Mode'}
                   </Box>
-                ),
+                )
               }}
             />
           </AppBar>
           <PageContent>
-            {authentication.user?.session ? <TimeOverviewPage /> : <LoginPage />}
+            {authentication.user?.session !== null && authentication.user?.session !== undefined ? <TimeOverviewPage /> : <LoginPage />}
```
Comment (SpiraTecGabelM) [inline src/App.tsx]: In Methode auslagern

File: src/api/base/login.ts [hunk 2]
```diff
@@ -16,38 +16,37 @@ const getSoapBody = (props: Credentials) => {
...
+          personID: session['ns2:personID'],
+          sessionID: session['ns2:sessionID']
         })
       })
       .catch((error) => {
-        let responseMessage: string | undefined = undefined
+        let responseMessage: string | undefined
 
-        if (error.data) {
+        if (error.data !== undefined) {
```
Comment (SpiraTecGabelM) [inline src/api/base/login.ts]: lodash isNil

Comment (SpiraTecGabelM) [reply src/api/base/login.ts]: Do not
import * as _ from 'lodash'
import {isNil} from 'lodash

Do
import isNil from 'lodash/isNil'

File: src/pages/login/LoginPage.tsx [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
+} from 'grommet'
+import { type Credentials } from '../../api/models'
+import { AuthContext } from '../../context/AuthContext'
 
-const LoginPage = () => {
+const LoginPage = (): JSX.Element => {
   const [isLoading, setIsLoading] = useState<boolean>(false)
-  const [username, setUsername] = useState<string>("")
-  const [password, setPassword] = useState<string>("")
+  const [username, setUsername] = useState<string>('')
```
Comment (SpiraTecGabelM) [inline src/pages/login/LoginPage.tsx]: introduce Blank constant in utils

export const Blank = ' '

File: src/pages/login/LoginPage.tsx [hunk 4]
```diff
@@ -60,7 +60,7 @@ const LoginPage = () => {
           />
         </Box>
       </Form>
-      {error && (
+      {error !== null && error !== undefined && (
```
Comment (SpiraTecGabelM) [inline src/pages/login/LoginPage.tsx]: use Method from above

File: src/App.tsx (hunk 1 above)
Comment (SpiraTecGabelM) [reply src/App.tsx]: Done

File: src/api/base/login.ts (hunk 2 above)
Comment (SpiraTecGabelM) [reply src/api/base/login.ts]: Done

File: src/pages/login/LoginPage.tsx (hunk 3 above)
Comment (SpiraTecGabelM) [reply src/pages/login/LoginPage.tsx]: Done

File: src/pages/login/LoginPage.tsx (hunk 4 above)
Comment (SpiraTecGabelM) [reply src/pages/login/LoginPage.tsx]: Done

**Review by EdgarKisman — APPROVED** [review]

**Pull request merged**